app/controllers/projects_controller.rb

```diff
@@ -59,8 +59,13 @@ def destroy
 
   def show
     @project = Project.find(params[:id])
-    @stories = @project.stories.by_position.includes(:estimates)
+    @q = @project.stories.by_position.includes(:estimates).ransack(params[:q])
```

Comment:
I'd say to call it `stories` as it was before, since it's querying stories, `q` or `query` are too generic

Reply:
In this case might be better to call it stories, but I have to add a call to `.result` at the end. Or we can just change it to `@query`, since I assign `@stories` in the line below

Reply:
I agree, `query.result` reads better than `stories.result` And then you assign to @stories like you said.